.evergreen/generate-buildvariants.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -o errexit  # Exit the script with error if any of the commands fail
+
+os=rhel80-small
+declare -a versions=("6.0" "7.0" "8.0")
+declare -a topologies=("server" "replica_set" "sharded_cluster")
```

Comment:
Let's just test against replica sets. Standalone servers don't support transactions so are not really useful for Hibernate testing. And sharded clusters should work exactly the same as replica sets from the perspective of Hibernate.